.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
Please add this to your local `~/.gitignore` instead.

```suggestion
```

Reply:
Yup, will do that. Thanks. :)